Client/src/Components/UserReview/UserReviewRow.tsx

```diff
@@ -0,0 +1,73 @@
+import { Box, Button, Grid, Typography } from "@material-ui/core"
+import { IApprovedDatasetModel, IFlaggedDatasetQuery } from "../../Models/Datasets/IApprovedDatasetModel"
+import React, { useState } from "react"
+import { approvedDataset, callRejectDataset, submitEditedDataset } from "../../Remote/Endpoints/DatasetEndpoint"
+
+import { DatasetModal } from "./DatasetModal"
+import { IDatasetModel } from '../../Models/Datasets/IDatasetModel'
+import { classStyles } from "../../appTheme"
+
+interface IProps {
+  dataset: IApprovedDatasetModel
+}
+
+export const UserReviewRow = (props: IProps) => {
+
+  const [open, setOpen] = useState(false)
+  const { dataset } = { ...props }
+
+  const reload = () => {
+    setTimeout(window.location.reload() as any, 200)
```

Comment:
this could have been avoided if the parent component who holds the datasets in the state would provide the handle Approve, handleDelete etc. then that parent component could just refetch unapproved datasets, set the state, boom, the page rerenders without reloading